fms_extras/models/paged_llama.py

```diff
@@ -713,6 +714,11 @@ def factory(**kwargs):
     _architecture_name, "micro", _llama_factory_factory(_micro_char_config)
 )
 models.register_model(_architecture_name, "7b", _llama_factory_factory(_7b_config))
+models.register_model(
+    _architecture_name,
+    "ibm.7b_instruct_lab",
```

Comment:
Since llama variants are names like '7b' it seems like we should do something like `7b.ibm_instruct_lab` or `7b.ibm.instruct_lab`?

I think what we're registering is the same as llama 7b and it's only the IBM + instruct_lab info that more granularly differentiates it, so I guess my impression is that the most general/common info should come first (and we don't use meta.7b for existing variants).